spec/pg_search_spec.rb

```diff
@@ -89,9 +89,9 @@ def self.call(arel, _context)
 
     it 'works for searching associations' do
       Arel.middleware.apply([SomeMiddleware]) do
-        salami1 = Salami.create!
-        salami2 = Salami.create!
-        salami3 = Salami.create!
+        salami1 = Salami.create! cracker: Cracker.create!
```

Comment:
I'm not sure why, but Salami needs a Cracker.

Reply:
🤣